### DISCUSSION TITLE
latest/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#740] [Docs]
# index

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/

**Comment by SkalskiP** [comment]
First-ever comment in the Supervision Documentation! 🔥

**Comment by joe-pires** [comment]
AttributeError: type object 'Detections' has no attribute 'from_yolov8'. Did you mean: 'from_yolov5'?

**Reply by SkalskiP** [reply]
Hi, @joe-pires! 👋🏻 `from_yolov8` got deprecated a few versions ago. Please use [`from_ultralytics`](https://supervision.roboflow.com/detection/core/#supervision.detection.core.Detections.from_ultralytics) instead.
